src/test/java/test/integration/cube/CubeSetupCommandTest.java

```diff
@@ -0,0 +1,154 @@
+package test.integration.cube;
+
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import test.integration.extension.AddPackage;
+import test.integration.support.ShellTestTemplate;
+
+import static test.integration.support.assertions.ForgeAssertions.assertThat;
+
+@RunWith(Arquillian.class)
+@AddPackage(ShellTestTemplate.PACKAGE_NAME)
+public class CubeSetupCommandTest extends ShellTestTemplate {
+
+    @Test
+    public void should_setup_arquillian_cube_for_docker() throws Exception {
+
+        shell().execute("arquillian-setup --standalone --test-framework junit")
+            .execute("touch src/test/resources/Dockerfile").execute("arquillian-cube-setup --type docker --file-path src/test/resources/ --docker-file-name Dockerfile");
+
+        assertThat(project).hasDirectDependency("org.arquillian.universe:arquillian-cube-docker").withType("pom").withScope("test");
+        assertThat(project).hasArquillianConfig().withExtension("docker").withProperty("definitionFormat", "CUBE");
```

Comment:
Don't use CUBE format, use always docker-compose format. You can embed it in this property as well.